### PR TITLE
ENG-1488 plugin deployment base name will be truncated if longer than…

### PR DESCRIPTION
**Pull request by firegloves**
… 32 chars, regardless of the plugin descriptor version

I added an application property to choose between truncate and not truncate.
To avoid injecting too many time the same thing in the app I used the workaround visible in the AppConfiguration: a non static method that injects the value and set the static variable. Not really clean but quick solution waiting for a real truncation management

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **firegloves**

If they are not already assigned, you can assign the PR to them by writing `/assign @firegloves` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-component-manager).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-component-manager/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-component-manager-ecmpr87**

File: src/main/java/org/entando/kubernetes/model/bundle/processor/PluginProcessor.java
Comment (firegloves) [inline src/main/java/org/entando/kubernetes/model/bundle/processor/PluginProcessor.java:99]: This override is useful to log also during the analysis phase

**Comment by entando-jx** [comment]
@firegloves: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 411c67d3bf9a07a63bfcbca78012c539eaf79127 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
@firegloves: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 1eede1b586969a06aa6655342e5cf5bb8af13f06 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-component-manager-ecmpr87**

**Review by ffleandro — APPROVED** [review]
Looks good to me

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando-k8s/entando-component-manager/pull/87#pullrequestreview-542872086" title="Approved">ffleandro</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-component-manager).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando-k8s/entando-component-manager/blob/master/OWNERS)~~ [ffleandro]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
